invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/InvitationsResults.js

```diff
@@ -36,9 +36,9 @@ export const InvitationsResults = ({ paginationOptions, currentResultsState }) =
             <ResultsPerPage
               values={paginationOptions.resultsPerPage}
               label={(cmp) => (
-                <Trans key="communitiesInvitationsResult" count={cmp}>
```

Comment:
I think we need more investigation into whether this isn't working. This is in the documentation as a recommended approach https://inveniordm.docs.cern.ch/develop/howtos/i18n/#singular-plural